src/css/app.css

```diff
@@ -72,6 +72,25 @@ html, body {
 .layer-tree-panel {
   max-height: 94vh !important;
 }
+.map-url-wrapper {
+	position: absolute;
+  top: 0;
+  width: 50%;
+  left: 25%;
+	height: 25px;
+	background: rgb(0, 188, 212);
```

Comment:
is it not better to use a material-ui Paper as root component so we get the theming from material-ui instead?